kikit/text.py

```diff
@@ -28,6 +28,7 @@ def kikitTextVars(board: pcbnew.BOARD, vars: Dict[str, str]={}) -> Dict[str, Any
     availableVars: Dict[str, Formatter] = {
         "date": Formatter(lambda: dt.datetime.today().strftime("%Y-%m-%d"), vars),
         "time24": Formatter(lambda: dt.datetime.today().strftime("%H:%M"), vars),
+        "timestamp": Formatter(lambda: dt.datetime.today().strftime("%Y%m%d_%H%M%S"), vars),
```

Comment:
The `date` and `time24` were added as they seemed generally useful. The timestamp option seems to me like something targeting a particular use case (as it uses a completely different format than `date` and `time24`).

I think it is better to provide individual variables for a year, month, day and hour, minutes and seconds. The users can then arrange them as they need.

Reply:
`time24` is great if you want to use it in text sections, however a colon is not a valid character in the file name both on windows and unix. This is why `timestamp` was proposed with that specific format; yeah providing the individual variables would be more flexible. I guess we will be keeping `date`/`time24` for backwards compatibility, right?

Reply:
Yes, I don't want to make a major release at the moment.